README.md

```diff
@@ -6,13 +6,19 @@ This is a microservice that allows user to create comments, edit comments and al
 
 For quick navigation:
 
+- [How it Works](#how-it-works)
```

Comment:
What is the reason for this?

Reply:
Will be added in a separate PR to give a big picture view of the entire micro-service. Give users/contributors an idea what the application does. I would have done that in this PR but I need to address the CommentModel first